app/views/spree/admin/payments/source_forms/_braintree.html.erb

```diff
@@ -0,0 +1,39 @@
+<fieldset data-id='credit-card' class="no-border-bottom">
+  <div class="field" data-hook="previous_cards">
+    <% if previous_cards.any? %>
+      <% previous_cards.each do |card| %>
+        <label><%= radio_button_tag :card, card.id, card == previous_cards.first %> <%= card.display_number %><br /></label>
+      <% end %>
+      <label><%= radio_button_tag :card, 'new', false, { id: "card_new#{payment_method.id}" } %> <%= Spree.t(:use_new_cc) %></label>
+    <% end %>
+  </div>
+
+  <div id="card_form<%= payment_method.id %>" data-hook>
+    <% param_prefix = "payment_source[#{payment_method.id}]" %>
+
+    <div class="clear"></div>
+
+    <div class="alpha four columns">
+      <div data-hook="card_name">
+        <div class="field">
+          <%= label_tag "card_name#{payment_method.id}", Spree::CreditCard.human_attribute_name(:name), class: 'required' %>
+          <%= text_field_tag "#{param_prefix}[name]", '', id: "card_name#{payment_method.id}", class: 'required fullwidth', maxlength: 19 %>
+        </div>
+      </div>
+    </div>
+
+    <%= javascript_include_tag "spree/backend/braintree/solidus_braintree", "data-turbolinks-track" => true, "crossorigin" => "anonymous" %>
```

Comment:
is this required since its also in the add_javascripts below? is it just added to the frontend, and this is the backend? can we add it to the backend in the same way as below?